Configuration/Geometry/python/GeometryDD4hepExtended2021_cff.py

```diff
@@ -5,7 +5,7 @@
 #
 # Ideal geometry, needed for simulation
 DDDetectorESProducer = cms.ESSource("DDDetectorESProducer",
-                                    confGeomXMLFiles = cms.FileInPath('DetectorDescription/DDCMS/data/cms-geometry-2021.xml'),
+                                    confGeomXMLFiles = cms.FileInPath('Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2021.xml'),
```

Comment:
@slomeo - as we have discussed it this morning, please, remove this file from the PR

Reply:
done